architects/demo/website_views.xml

```diff
@@ -0,0 +1,181 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="homepage" model="ir.ui.view">
+        <field name="name">Home</field>
+        <field name="key">architects.homepage</field>
+        <field name="type">qweb</field>
+        <field name="website_id" ref="website.default_website"/>
+        <field name="arch" type="xml">
+            <t name="Homepage" t-name="website.homepage">
+                <t t-call="website.layout">
+                    <t t-set="pageName" t-value="'homepage'"/>
+                    <div id="wrap" class="oe_structure oe_empty">
+                        <section class="s_banner parallax s_parallax_is_fixed o_cc o_cc3 o_colored_level" data-scroll-background-ratio="1" data-oe-shape-data="{&quot;shape&quot;:&quot;web_editor/Origins/18&quot;,&quot;flip&quot;:[&quot;x&quot;]}" data-snippet="s_banner" data-name="Banner" style="background-image: none;">
+                            <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/website.s_banner_default_image'); background-position: 50% 0;"/>
+                            <div class="o_we_bg_filter bg-black-50"/>
+                            <div class="o_we_shape o_web_editor_Origins_18" style="background-image: url('/web_editor/shape/web_editor/Origins/18.svg?c1=o-color-2&amp;flip=x'); background-position: 50% 50%;"/>
+                            <div class="container">
+                                <div class="row s_nb_column_fixed">
+                                    <div class="col-lg-7 h-100 pt160 pb160 o_colored_level" data-name="Box" style="">
+                                        <h1>
+                                            <b>Architectural Office</b>
+                                        </h1>
+                                        <h1>
+                                            <b>Near You.</b>
+                                        </h1>
+                                        <pre>Designs and carries out contemporary and unique projects throughout the world</pre>
+                                        <p class="lead">
+                                            <br/>
+                                        </p>
+                                        <a class="btn btn-primary mb-2" href="/contactus" data-bs-original-title="" title="">Reach out</a>
+                                    </div>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+                </t>
+            </t>
+        </field>
+    </record>
+    <record id="contactus" model="ir.ui.view">
```

Comment:
From what I understand, the only change here is to create an opportunity from a click on submit... Couldn't we reduce the diff then, and use the default contactus page as a basis?